operator/api/v1alpha1/httpscaledobject_types.go

```diff
@@ -41,6 +77,22 @@ const (
 	Unknown HTTPScaledObjectCreationStatus = "Unknown"
 )
 
+// Condition to store the condition state
+type HTTPScaledObjectCondition struct {
```

Comment:
I am curious, do you think that it is doable to use core KEDA Condition Type for this: https://github.com/kedacore/keda/blob/main/api/v1alpha1/condition_types.go#L20 ?

Reply:
Do you mean using as a module instead of copying?

Reply:
@zroubalik I think this was autogenerated like the KEDA core condition was. Better to stick with this?

Reply:
Yeah I meant using a module. But I missed that you are using the autogeneration for this, in the KEDA core we added conditions when there wasn't autogeneration capabilities for this.

So I guess that you can keep the autogenerated stuff, if it is more convenient for you:)